console/src/app/modules/nav/nav.component.html

```diff
@@ -212,7 +212,46 @@
 
 <ng-template #shortcutKeyRef>
   <ng-container *ngIf="(isHandset$ | async) === false">
-    <span class="fill-space"></span>
+    <ng-template cnslHasRole [hasRole]="['iam.read']">
```

Comment:
this box doesn't work at all
![image](https://user-images.githubusercontent.com/27845747/221163493-86fcb980-ed37-404e-aa1e-2957ab733126.png)

Reply:
removed